.github/workflows/ci.yml

```diff
@@ -8,4 +8,65 @@ jobs:
     steps:
       - uses: actions/checkout@v1
       - uses: actions/setup-node@v1
-      - run: yarn && yarn lint
+        with:
+          node-version: '14'
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Run lint
+        run: yarn lint
+
+  Test:
```

Comment:
Maybe `E2E tests` here? Because we have an custom plugin and it has unitary tests so we have two category of tests on this repository.

Reply:
sure